src/components/LoginRegister/ExampleLoginUtility.js

```diff
@@ -0,0 +1,47 @@
+// comment on why this file exists
```

Comment:
so, you'll see a bit lower that I really don't think you should have utilities like this (especially network requests) in a component directly. 

Ideally they are in a custom hook (for example, a [useFetch hook like this article shows](https://blog.bitsrc.io/fetching-data-in-react-using-hooks-c6fdd71cb24a), but I think that may be a little advanced to get into based on where you're at. to match your current implementation, I'd at least pull the function out into it's own file. There's a few big reasons for this:

1. You can more easily test the network request function.
2. It separates the concerns out of the component. You want your components to be as simple as possible and (ideally) not have to worry about the details of a network request.
3. It allows you to write the network request code in a more reusable way. Note the `onSuccess` and `onFailure` here - it would allow you to use this `fetchUser` function on multiple components, and let them tell the function what to do when a success or failure happens.